sycl/include/CL/sycl/accessor.hpp

```diff
@@ -243,16 +243,23 @@ template <typename DataT, int Dimensions, access::mode AccessMode,
 class image_accessor
 #ifndef __SYCL_DEVICE_ONLY__
     : public detail::AccessorBaseHost {
-  size_t MImageSize;
   size_t MImageCount;
+  size_t MImageSize;
 #else
 {
-  /*
-  // TODO: Define the datatype here based on Dimensions, AccessMode and
-  AccessTarget.
-  __ocl_image[Dim]d[array/non_array]_[AM]_t MImage;
-  __init(__ocl_imagexx_t Image) { MImage = Image; }
-  */
+
+  using OCLImageTy = typename detail::opencl_image_type<Dimensions, AccessMode,
+                                                        AccessTarget>::type;
+  OCLImageTy MImageObj;
+  char
+      padding[(sizeof(detail::AccessorBaseHost) +
```

Comment:
Please, follow way of naming variables in the file.

Reply:
I didn't get the comment. 
The other classes use 'padding' in this file.

Reply:
Updated the name.